checks.d/mysql.py

```diff
@@ -32,28 +32,241 @@
     'Com_insert': ('mysql.performance.com_insert', RATE),
     'Com_update': ('mysql.performance.com_update', RATE),
     'Com_delete': ('mysql.performance.com_delete', RATE),
+    'Com_replace': ('mysql.performance.com_replace', RATE),
+    'Com_load': ('mysql.performance.com_load', RATE),
     'Com_insert_select': ('mysql.performance.com_insert_select', RATE),
     'Com_update_multi': ('mysql.performance.com_update_multi', RATE),
     'Com_delete_multi': ('mysql.performance.com_delete_multi', RATE),
     'Com_replace_select': ('mysql.performance.com_replace_select', RATE),
+    # Connection Metrics
+    'Connections': ('mysql.net.connections', RATE),
+    'Max_used_connections': ('mysql.net.max_connections', RATE),
```

Comment:
Indeed, `mysql.net.max_connections` should be a gauge and `mysql.net.connections` should be a monotonic count (ever increasing)